source/scale/scale-to-200000-users.rst

```diff
@@ -0,0 +1,64 @@
+  - From Mattermost v10.4, Mattermost Enterprise customers can configure `Redis <https://redis.io/>`_ (Remote Dictionary Server) as an alternative cache backend. Using Redis can help ensure that Mattermost remains performant and efficient, even under heavy usage. See the :ref:`Redis cache backend <configure/environment-configuration-settings:redis cache backend>` configuration settings documentation for details.
+  - While the following Elasticsearch specifications may be more than sufficient for some use cases, we have not extensively tested configurations with lower resource allocations for this user scale. If cost optimization is a priority, admins may choose to experiment with smaller configurations, but we recommend starting with the tested specifications to ensure system stability and performance. Keep in mind that under-provisioning can lead to degraded user experience and additional troubleshooting effort.
+
+Requirements
+------------
+
++------------------------+-----------+----------------+-----------------------+
+| **Resource Type**      | **Nodes** | **vCPU/        | **AWS Instance**      |
+|                        |           | Memory (GiB)** |                       |
++========================+===========+================+=======================+
+| Mattermost Application | 14        | 16/32          | c7i.4xlarge           |
++------------------------+-----------+----------------+-----------------------+
+| RDS Writer             | 1         | 16/128         | db.r7g.4xlarge        |
++------------------------+-----------+----------------+-----------------------+
+| RDS Reader             | 6         | 16/128         | db.r7g.4xlarge        |
++------------------------+-----------+----------------+-----------------------+
+| Elasticsearch cluster  | 4         | 8/64           | r6g.2xlarge.search    |
++------------------------+-----------+----------------+-----------------------+
+| Proxy                  | 4         | 32/128        | m6in.8xlarge           |
++------------------------+-----------+----------------+-----------------------+
+| Redis                  | 1         | 8/32          | cache.m7g.2xlarge      |
++------------------------+-----------+----------------+-----------------------+
+
+Lifetime storage
+----------------
+
+.. include:: ../scale/lifetime-storage.rst
+  :start-after: :nosearch:
+
+Estimated storage per user, per month
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../scale/estimated-storage-per-user-per-month.rst
+  :start-after: :nosearch:
+
+Example
+~~~~~~~
+
+A 100000-person team with medium usage (with a safety factor of 2x) would require between 10.56TB :sup:`1` and 52.8TB :sup:`2` of free space per annum.
+
+:sup:`1` 100000 users * 5 MB * 12 months * 2x safety factor
+
+:sup:`2` 100000 users * 25 MB * 12 months * 2x safety factor
+
+We strongly recommend that you review storage utilization at least quarterly to ensure adequate free space is available.
```

Comment:
Should this example be updated to 200k?

Reply:
Yes please: the 100K inline references and the final calculation of free space

Reply:
Will do